plasmapy/particles/particle_class.py

```diff
@@ -2,6 +2,13 @@
 
 from __future__ import annotations
 
+from io import StringIO
+from typing import Any, Optional, Union
+
+from astropy.units.quantity import Quantity
+
+from plasmapy.particles.particle_collections import ParticleList
```

Comment:
Ah. This automatically added line seems to be what's causing problems.  Need to add an `if typing.TYPE_CHECKING` style flag in here.  Probably.